Packages/Tracking/Core/Runtime/Scripts/Utils/HandUtils.cs

```diff
@@ -379,6 +379,13 @@ public static float GetFingerStrength(this Hand hand, int finger)
             return Vector3.Dot(hand.Fingers[finger].Direction, -hand.DistalAxis()).Map(-1, 1, 0, 1);
         }
 
+        /// <summary>
+        /// Returns the Chirality of the hand
+        /// </summary>
+        public static Chirality GetChirality(this Hand hand) {
```

Comment:
can this brace go on the next line to match the rest of the file please? :)

Reply:
done